cmd/server/main.go

```diff
@@ -533,3 +565,56 @@ func initNamespace(config config.Configuration, namespaces ...namespace.Handler)
 	slog.Info("default namespace created")
 	return namespaceManager, nil
 }
+
+func initPGClients(ctx context.Context, config config.PostgresConfig) (*struct {
+	driver                 *entDialectSQL.Driver
+	entitlementDBClient    *entitlementdb.Client
+	productcatalogDBClient *productcatalogdb.Client
+	creditDBClient         *creditdb.Client
+}, error) {
+	if err := config.Validate(); err != nil {
+		return nil, fmt.Errorf("invalid postgres config: %w", err)
+	}
+	driver, err := entutils.GetPGDriver(config.URL)
```

Comment:
Is this going to open connection pools per connectors or share the connection pool between connectors?

Reply:
`*sql.DB` and `*driver.Connector` manage pooling, thats part of driver, so on, its a single pool